docs/commands/zed.md

```diff
@@ -384,13 +384,15 @@ case but this design allows different workloads like these to be custom tuned.
 
 The `zed` command is structured as a primary command
 consisting of a large number of interrelated sub-commands, similar to the
-[docker](https://docs.docker.com/engine/reference/commandline/cli/)
-or [kubectl](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands)
+[`docker`](https://docs.docker.com/engine/reference/commandline/cli/)
+or [`kubectl`](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands)
 commands.
 
-The following sections describe each of the available commands, but built-in
-help is also available:
-* `zed -h` with no args displays a list of zed commands.
+The following sections describe each of the available commands and highlight
+some key options. Built-in help shows all commands and options that can be
+invoked as follows:
```

Comment:
1. The new sentence reads oddly to me.
2. Prevailing style in here is two spaces between sentences.
```suggestion
some key options. Built-in help shows the commands and their options:
```